osu.Game/Rulesets/Mods/ModHidden.cs

```diff
@@ -58,5 +58,9 @@ public ScoreRank AdjustRank(ScoreRank rank, double accuracy)
         protected virtual void ApplyHiddenState(DrawableHitObject hitObject, ArmedState state)
         {
         }
+
+        protected virtual void ApplySpecialHiddenState(DrawableHitObject hitObject, ArmedState state)
+        {
+        }
```

Comment:
Does this need to exist in the base? Seems like it could be local to the osu! implementation, and it's a bit weird having a *special* hidden state that isn't really used anywhere else.

Reply:
no matter where this exists, if you are going to call something "special" it *needs* xmldoc